lib/IonDid.ts

```diff
@@ -0,0 +1,180 @@
+import DidDocumentKey from './DidDocumentKey';
+import Encoder from './Encoder';
+import ErrorCode from './ErrorCode';
+import IonError from './IonError';
+import JsonCanonicalizer from './JsonCanonicalizer';
+import JwkEs256k from './models/JwkEs256k';
+import Multihash from './Multihash';
+import PublicKeyModel from './models/PublicKeyModel';
+import SdkConfig from './SdkConfig';
+import ServiceEndpointModel from './models/ServiceEndpointModel';
+
+/**
+ * Class containing DID related operations.
+ */
+export default class IonDid {
+  /**
+   * Creates a long-form DID.
+   * @param didDocumentPublicKeys Public keys to be included in the resolved DID Document.
+   * @param serviceEndpoints  Service endpoints to be included in the resolved DID Document.
+   */
+  public static createLongFormDid (input: {
```

Comment:
I fear this is the wrong method to expose, 'coz I think about how would people then use our SDK to register a long form? Maybe the right method here is to create a create operation and let long-form DID derive (a property) from that?!

Reply:
I see the point. This is really convenient though...